app/views/layouts/_header.html.erb

```diff
@@ -5,40 +5,60 @@
     <% end %>
   </div>
   <div id='user-info'>
-    <span id='logout_area'>
-      <span id='instructor_logged_in_as'>
+    <ul>
+      <li id="user-info-dropdown">
+        <div class = "dropdown">
+        <span id='instructor_logged_in_as'>
         <% if allowed_to?(:role_is_switched?) %>
           <strong><%= @real_user.display_name %></strong>
```

Comment:
Let's put this name in parentheses (previously, the `@current_user` as in parentheses), to maintain a split between them.

You should test out the UI for the dropdown after switching roles to see this. *ALSO*, please add an icon for the "Cancel role switch" action. (Use the same icon that currently appears on master.)